actor.go

```diff
@@ -152,13 +152,18 @@ type ActorSpeak struct {
 	ActorName string
 	Text      string
 	Delay     time.Duration
+	Color     Color
 }
 
 func (cmd ActorSpeak) Execute(app *App, done Promise) {
 	if cmd.Delay == 0 {
 		cmd.Delay = DefaultActorSpeakDelay
 	}
 
+	if cmd.Color == rl.Blank {
+		cmd.Color = rl.White
+	}
+
```

Comment:
So what? You are not doing anything with this color passed by the command.

Reply:
OMG, sorry, I forgot the most important part! 🤦 
Done!